serverless/knative_eventing/serverless-pingsource.adoc

```diff
@@ -0,0 +1,33 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-pingsource"]
+= Using PingSource
+:context: serverless-pingsource
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+PingSource is used to periodically send ping events with a constant payload to an event consumer.
+
+PingSource can be used to schedule sending events, like a timer, as shown in the example:
```

Comment:
You can use PingSource to schedule...

_like a time_. Does this mean that PingSource can be used the same as a timer? _similar to a timer_
Or, is a timer one of the sending events? _such as a timer_